### PR TITLE
API-7501: v3.3+: document new property configuration endpoints

### PR DESCRIPTION
**Pull request by litwinow**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7501-properties-configuration)
- [Jira](https://livechatinc.atlassian.net/browse/API-7501)

## 📓 Description

New endpoints for properties' configuration management.

## ✅ Checklist (for API docs)

- Changelog ✅ 
- API versions ✅ 
- Web & RTM ✅ 
- **Table:** Available methods/webhooks/pushes ✅ 
- **Table:** Specifics ✅ 

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

## 👷 Type of change (remove not needed)

New content, removal of deprecated content.

### Content

- New content
  
### Features (for the website)

- New feature

### CODE REVIEW

File: content/management/configuration-api/v3.3/index.mdx [hunk 1]
```diff
@@ -1774,20 +1770,27 @@ curl -X POST \
...
 
 #### Request
 
-| Parameter | Required | Data type | Notes                                                                                 |
-| --------- | -------- | --------- | ------------------------------------------------------------------------------------- |
-| `all`     | No       | `bool`    | If set to `true`, it returns all properties within a given license; default: `false`. |
+| Parameter         | Required | Data type  | Notes                                                                     |
+| ----------------- | -------- | ---------- | ------------------------------------------------------------------------- |
+| `name`            | Yes      | `string`   | Property name                                                             |
+| `owner_client_id` | Yes      | `string`   | **Client Id** which owns the property; must be owned by your organization |
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1789]:
```suggestion
| `owner_client_id` | Yes      | `string`   | **Client Id** that owns the property; must be owned by your organization |
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 2]
```diff
@@ -1797,15 +1800,112 @@ curl -X POST \
...
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Publish Property
+
+Publishes a private property. A published property cannot be unpublished and cannot be unregistered. Published properties can be accessed by tokens belonging to any **Client Id**.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1824]:
```suggestion
Publishes a private property. A published property cannot be unpublished or unregistered. Published properties can be accessed by tokens belonging to any **Client Id**.
```

**Review by ale-kacperczyk — APPROVED** [review]

File: content/management/changelog/index.mdx [hunk 3]
```diff
@@ -38,6 +38,13 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**List Group Properties**](/management/configuration-api/v3.3/#list-group-properties) method had the `group_id` parameter renamed to `id`.
 - The [**Delete Group Properties**](/management/configuration-api/v3.3/#delete-group-properties) method had the `group_id` parameter renamed to `id`.
+- Properties configuration is now owned by **Client Id**, not by **License**, which resulted in the following changes:
```
Comment (janowsiany) [inline content/management/changelog/index.mdx:41]: Shouldn't we refer to **Integration** and not **Client Id**? here and in all other places

File: content/management/changelog/index.mdx [hunk 4]
```diff
@@ -38,6 +38,13 @@ Since the migration is still ongoing, different parts will be successively moved
...
+  - The **Register Properties** method was removed.
+  - The **List Registered Properties** method was removed.
+    - [**Register Property**](/management/configuration-api/v3.3/#register-property)
+    - [**Unregister Property**](/management/configuration-api/v3.3/#unregister-property)
+    - [**Publish Property**](/management/configuration-api/v3.3/#publish-property)
+    - [**List Properties**](/management/configuration-api/v3.3/#list-properties)
```
Comment (janowsiany) [inline content/management/changelog/index.mdx:47]: Is spacing ok here? Looks like it is nested. Also, in the first two points, we used complete sentences, why no here?

**Review by janowsiany — CHANGES_REQUESTED** [review]

File: content/management/changelog/index.mdx (hunk 4 above)
Comment (ale-kacperczyk) [reply content/management/changelog/index.mdx:47]: Good catch :) I Accidentally deleted one sentence. It was supposed to be a double-nested list.

File: content/management/changelog/index.mdx [hunk 5]
```diff
@@ -38,6 +38,13 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**List Group Properties**](/management/configuration-api/v3.3/#list-group-properties) method had the `group_id` parameter renamed to `id`.
 - The [**Delete Group Properties**](/management/configuration-api/v3.3/#delete-group-properties) method had the `group_id` parameter renamed to `id`.
+- Properties configuration is now owned by **Client Id**, not by **License**, which resulted in the following changes:
+  - The **Register Properties** method was removed.
+  - The **List Registered Properties** method was removed.
```
Comment (ale-kacperczyk) [inline content/management/changelog/index.mdx:43]:
```suggestion
  - The **List Registered Properties** method was removed.
  - There are new methods:
```

File: content/management/changelog/index.mdx (hunk 3 above)
Comment (ale-kacperczyk) [reply content/management/changelog/index.mdx:41]: We've been using both, I guess. Perhaps we could write Client Id (Integration).

File: content/management/changelog/index.mdx [hunk 6]
```diff
@@ -38,6 +38,14 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**List Group Properties**](/management/configuration-api/v3.3/#list-group-properties) method had the `group_id` parameter renamed to `id`.
 - The [**Delete Group Properties**](/management/configuration-api/v3.3/#delete-group-properties) method had the `group_id` parameter renamed to `id`.
+- Properties configuration is now owned by **Client Id**, not by **License**, which resulted in the following changes:
```
Comment (ale-kacperczyk) [inline content/management/changelog/index.mdx:41]:
```suggestion
- Properties configuration is now owned by **Client Id** (integration), not by **License**, which resulted in the following changes:
```

**Review by janowsiany — APPROVED** [review]

**Pull request merged**